skfda/exploratory/visualization/_boxplot.py

```diff
@@ -249,24 +245,49 @@ class Boxplot(FDataBoxplot):
 
     """
 
-    def __init__(self, fdatagrid, depth_method=ModifiedBandDepth(), prob=[0.5],
-                 factor=1.5):
+    def __init__(
+        self,
+        fdatagrid: FDataGrid,
```

Comment:
Is this necessarily an `FDataGrid`?

Reply:
i dont know, it was how it was before

Reply:
Well, for now we only have depths for `FDataGrid` but I do not think that it is necessarily restricted to that.